### ISSUE TITLE
Idea

### ISSUE THREAD

**Issue by Shade7282** [Prodigy-Hacking/ProdigyMathGameHacking#1340]
switch from a persistent background page to an event page that is unloaded when an extension is not used.
so therefore you maybe (i dont code so) people can make a book mark and use the extension via bookmark (again i dont code im not even sure its possible with this extension)

**Comment by afkvido** [comment]
It is possible to load the bundle with JavaScript i think

**Comment by 1603191909** [comment]
the book mark would be very large?

**Comment by 1603191909** [comment]
actually might not be possible, redirects are required

**Comment by Shade7282** [comment]
well obviously it wouldnt be the full thing the ideo is just to add the main stuff people even get for like give all items/weapon/armor and set level and maybe insta win battle

**Comment by afkvido** [comment]
> the book mark would be very large?

No, if the bookmark fetched the bundle with javascript with `fetch`.

the bundle wouldnt be in the bookmark

**Comment by afkvido** [comment]
> so therefore you maybe (i dont code so) people can make a book mark and use the extension via bookmark (again i dont code im not even sure its possible with this extension)

You can't replace the extension.

> actually might not be possible, redirects are required

**Comment by LeoBadeaux** [comment]
> > the book mark would be very large?
> 
> No, if the bookmark fetched the bundle with javascript with `fetch`.
> 
> the bundle wouldnt be in the bookmark

While you could create a somewhat limited version of the hacks and have it as a bookmarklet, you won't be able to remotely update it or anything with fetch due to CORS.

**Comment by afkvido** [comment]
I thought of that when posting my second comment, but if it was possible, it would just fetch the bundle.

**Comment by afkvido** [comment]
this issue should be closed

**Comment by PatheticMustan** [comment]
> people can make a book mark and use the extension via bookmark
because of the way we inject hacks, this is not possible.
